src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -680,15 +686,17 @@ def __call__(self):
 
         try:
             origCores = int(getattr(self.step.data.application.multicore, 'numberOfCores', 1))
+            eventStreams = int(getattr(self.step.data.application.multicore, 'eventStreams', 0))
             resources = {'cores': origCores}
-            resizeResources(resources)
+            if not eventStreams:  # Don't know how to handle resizing jobs with different nThreads and nStreams
```

Comment:
Giving this a second thought. I think it's more dangerous not to resize than perform a non-ideal resize. Consider the following: a job originally requesting 8 cores (but resizable from 4 to 8) matches against a 4 cores slot. If we don't resize it, this job would spawn the original 8 cores and then the worker node is in trouble.

For the moment, how about we set it to 0 in case `origCores != numCores`?